lib/absinthe/execution/variable.ex

```diff
@@ -132,6 +140,26 @@ defmodule Absinthe.Execution.Variable do
     build_map_value(value, real_type, type_stack, var_ast, meta)
   end
 
+  defp inner_list_values(raw_values, inner_type, type_stack, var_ast, meta) do
+    inner_list_values(raw_values, inner_type, type_stack, var_ast, [], meta)
+  end
+
+  defp inner_list_values([], _, _, _, acc, meta), do: {:lists.reverse(acc), meta}
+  defp inner_list_values([raw_value | rest], inner_type, type_stack, var_ast, acc, meta) do
+    case build_map_value(raw_value, inner_type, type_stack, var_ast, meta) do
+      {:ok, nil, meta} ->
+        inner_list_values(rest, inner_type, type_stack, var_ast, acc, meta)
+
+      {:ok, value, meta} ->
+        inner_list_values(rest, inner_type, type_stack, var_ast, [value | acc], meta)
+
+      {:error, meta} ->
+        # No need to put an error here because which ever build_map_value clause
+        # failed should have added it already.
+        inner_list_values(rest, inner_type, type_stack, var_ast, acc, meta)
+    end
+  end
```

Comment:
I need to try to consolidate these walk helpers but this gets stuff done for now.